macro/Version.py

```diff
@@ -36,19 +36,15 @@
 Dependencies = []
 
 # configure the active set of distros
-from macroutils import distro_names as distros
+from macroutils import distro_names, distro_names_hidden
 
-if 'boxturtle' in distros:
-    distros.remove('boxturtle')
-if 'unstable' in distros:
-    distros.remove('unstable')
+distros = [d for d in distro_names if d not in distro_names_hidden]
```

Comment:
Ok, I think I misunterpreted what you meant when you said 

> but if you search for e.g. boxturtle in this repo there are numerous locations where specific distro names are being ignored

could you clarify how this ties to declaring the list of hidden distros in macroutils ? do you mean creating another list for the subset of distros ignored in version?